### ISSUE TITLE
Question about ECCV paper

### ISSUE THREAD

**Issue by LightingMc** [keenon/InferBiomechanics#17]
Hey, hope you are doing well. If any of you could kindly answer the questions below, it will help me a lot @keenon @rishiagarwal2000 @aidanchandra @janellekaneda 

I wanted to know the exact settings used for the CVPR paper.

![image](https://github.com/user-attachments/assets/7ff3d709-5c1b-47f7-bacd-6d6ccc4a4006)

1. Why is Vanderzee not included in the dataset? Did the MLP results shown in the CVPR paper also not include Vanderzee as well?
[AddBiomechanicsDataset.py](https://github.com/keenon/InferBiomechanics/blob/main/src/data/AddBiomechanicsDataset.py)
`                    if file.endswith(".b3d") and "vander" not in file.lower():
                        self.subject_paths.append(os.path.join(root, file))
`

2. The paper states 37 DOFs. Does that mean you only trained with the "With_Arm" data because it had 37 DOFs? "No arm" had 23 DOF. 
Fig1. Text
![image](https://github.com/user-attachments/assets/7ba35171-512e-49f0-be7e-a4a4e3c29d42)

3. What was the number of epochs, optimizer, and learning rate for the MLP training? The configuration files had a lot of different settings and it was hard to tell which were the ones used for the paper.

4. The paper states using only Accelerations as the inputs for the MLP. But the code seems to be using all of the available kinematics data, ex. pos, vel, acc, angles, root_hist stuff etc.
